interactive-tests/default.rb

```diff
@@ -13,3 +23,11 @@
 end
 sleep 1.5
 $test.save_screenshot
+
+close_dialog_size = $test.driver.find_elements(:css=>'.about-dialog > .ui-dialog-titlebar > .ui-dialog-titlebar-close').size #checks to see if there is an about dialog
+if close_dialog_size > 0 && visible #some interactives have hidden about dialogs so have to check if they are visible
+    puts "about box is visible"
+  $test.driver.find_element(:css=>'.about-dialog > .ui-dialog-titlebar > .ui-dialog-titlebar-close').click
+  sleep 1.5
+  $test.save_screenshot
+end
```

Comment:
Note that there's already a function that closes About dialog:
https://github.com/concord-consortium/lab-selenium-tests/blob/master/lib/test_api.rb#L38-L46

I guess it could be used (you removed calls to it below). Or if you need to check if the dialog is present first, I guess we could add a new one there, something like `close_about_dialog_if_present`.